docs/user-guide/annotations.md

```diff
@@ -120,26 +119,28 @@ Click on <img src="../img/button/apply_button.jpg" class="ms-docbutton"/> to app
 Managing Annotations
 --------------------
 
-You can manage anytime your annotations through the *Annotation Viewer*. Click on some annotation to display it.
+Once annotations are added to the [TOC](toc.md), the user can **Manage** it by clicking to <img src="../img/button/edit_button.jpg" class="ms-docbutton" /> button from the TOC toolbar and the *Main Annotations panel* will be open.
 
-<img src="../img/annotations/annotation_toolbar.jpg" class="ms-docimage" style="max-width:500px;"/>
+<img src="../img/annotations/annotations_main_panel.jpg" class="ms-docimage" style="max-width:500px;" />
 
-* You can edit your annotations by clicking the geometry card from list of geometries of the annotation. It allows you to change the geometry and/or the information related to the annotations. See the gif below to better understand how:
+From it, the editor is allowed to:
```

Comment:
here you use the word editor and in other places the word user, can you align these to editor or user?